ui/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "test": "npm-run-all lint",
+    "test": "echo TODO",
```

Comment:
`npm-run-all test:unit` even if it does not work

Reply:
done